content/collections/destination-catalog/en/hubspot-event-streaming-v2.md

```diff
@@ -52,6 +53,19 @@ Keep these things in mind when sending events to HubSpot:
 7. Under *Map properties to destinations*, select the Event Properties you would like to send. The HubSpot identifier for this object must be at least one of the following: `User Token`, `Email` or `Object ID`.
 8. Under *Select additional properties*, select any extra event and user properties you want to send to HubSpot. If you don't select any properties here, Amplitude doesn't send any.
 9. When finished, enable the destination and click *Save*.
+10. Click *Create in HubSpot* if there are events and/or properties that are missing.
+11. Click *Continue*, verify your connection, and click *Finish*.
+
+### Troubleshooting
+If you manually created and then deleted a custom event in HubSpot, the internal name is locked and can't be reused. Here's a workaround:
+
+1. In Amplitude Data, navigate to *Events > Custom Events*.
+2. Click *Create custom event*.
+3. Name the custom event something similar to the original, but not the same (for example, if you deleted `user_signup`, use `user_signup_v2`). This ensures Amplitude can create the event in HubSpot without naming conflicts.
+4. Configure the custom event to trigger on the deleted event.
+5. Update your HubSpot event stream to send this new custom event instead of the original one.
+6. Click *Create in HubSpot* to define the custom event in HubSpot.
```

Comment:
May be confusing to skip steps like this. Could just suggest they complete the setup flow again with the new custom event.